libs/cart/src/effects/cart-item.effects.ts

```diff
@@ -65,29 +74,45 @@ export class DaffCartItemEffects<
   add$ = this.actions$.pipe(
     ofType(DaffCartItemActionTypes.CartItemAddAction),
     switchMap((action: DaffCartItemAdd<U>) =>
-      this.driver.add(
-        this.storage.getCartId(),
-        action.input
-      ).pipe(
-        map((resp: V) => new DaffCartItemAddSuccess(resp)),
+      combineLatest([
+				this.store.pipe(
+					select(getDaffCartItemEntitiesSelectors<V, X, T>().selectAllCartItems),
+					take(1)
+				),
```

Comment:
Do you think its okay to use the facade instead of the store here?

Reply:
Though I don't think it's that big of a deal, I think it's probably better practice not to use the facade, because the selectors are more essential than the facade. It just seems like a bad dependency chain, even though it would work.

^I wrote that, then I thought that there is really no dependency relationship between effects and the facade. That dependency is between the selectors and the facade so I think it should be ok to use the facade.

Reply:
Its not a big deal either way I just think the facade is cleaner to interact with.

Reply:
now using the facade. The tests were throwing weird errors when I used the `DaffCartTestingModule`, so I just mocked it out instead and it works. Not sure why

Reply:
Its because of the injection token mismatch between `src` and `dist`. I rebased to get my recent spec tsconfig change and changed all of the imports to `@daffodil/cart` and the testing module works now.

I also exported the debounce timer injection token, which wasn't being exported before. I only noticed because I was importing from `@daffodil/cart`, that change is already paying off :D